.github/workflows/ci.yml

```diff
@@ -34,7 +34,13 @@ jobs:
           -DCLIENT_IDENTIFIER="ci-identifier" \
           -DTHING_NAME="thing-name" \
           -DS3_PRESIGNED_GET_URL="get-url" \
-          -DS3_PRESIGNED_PUT_URL="put-url"
+          -DS3_PRESIGNED_PUT_URL="put-url" \
+          -DCLAIM_CERT_PATH="cert/path" \
+          -DCLAIM_PRIVATE_KEY_PATH="key/path" \
```

Comment:
Perhaps these should be commandline arguments to the demo?

Reply:
This would break convention with the rest of our demos where we configure the parameters either through modifying the demo_config.h or with cmake